### ISSUE TITLE
Add SVG suggestions to complement icon fonts

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#18588]
http://v4-alpha.getbootstrap.com/migration/#components should mention some SVG alternatives, rather than (or at least, in addition to) icon fonts.

ref:
https://speakerdeck.com/ninjanails/death-to-icon-fonts
http://blog.cloudfour.com/seriously-dont-use-icon-fonts/

**Comment by cvrebert** [comment]
> https://speakerdeck.com/ninjanails/death-to-icon-fonts?slide=19

Seems silly that the browser isn't falling back from OpenDyslexic to the icon font, given that all the icon codepoints should be in the Unicode Private Use Area where OpenDyslexic shouldn't have any glyphs. I guess the font override being used just sets something like `font-family: OpenDyslexic !important;` rather than prepending OpenDyslexic to the `font-family` lists?

**Comment by thecristen** [comment]
[Glyph](http://glyph.smarticons.co/) offers SVG icons, and [Linea](http://linea.io/) and [Open Iconic](https://useiconic.com/open/) offer SVG versions of their icons too.

**Comment by bastienmoulia** [comment]
Octicons is already in the list and use SVG instead of font.

**Comment by myorangeca** [comment]
Do we really need to add icons when we have alternatives like font-awesome? I think we should focus more on simplifying the core...

**Comment by bastienmoulia** [comment]
@myorangeca It is only to propose alternatives to Glyphicons in the documentation, not in the core.

**Comment by thiagotomasi** [comment]
@patrickhlauke are you on it? Can I help you with something?

**Comment by thiagotomasi** [comment]
@mdo, should I we proceed with this issue or not? 😄
I believe I can help with this.

**Comment by mdo** [comment]
A list was added in 672fde1a0235fdcf9cd324902f3ff9d874e9a9de. Closing for now.
